docs/src/main/asciidoc/compose-dev-services.adoc

```diff
@@ -664,7 +664,7 @@ This allows services to be reused by other applications or subsequent runs of th
 
 You can also configure the timeout for stopping services with the `quarkus.compose.devservices.stop-timeout` property.
 After the timeout, Compose Dev Services will forcefully stop the services.
-The default timeout is chosen deliberately short to 1 second for fast cleanup, but you can increase it as needed:
+The default timeout is deliberately chosen to be 1 second for fast cleanup, but you can increase it as needed:
```

Comment:
I like the 'short' because it makes the 'why' clearer; 1 second isn't a magic number, it's just short. What about 

```suggestion
The default timeout is deliberately chosen to be short (1 second) for fast cleanup, but you can increase it as needed:
```